projects/taiga-schematics/schematics/ng-add/tests/ng-add.spec.ts

```diff
@@ -110,7 +110,7 @@ describe('ng-add', () => {
             "options": {
               "main": "test/main.ts",
             "styles": [
-              "node_modules/@taiga-ui/core/styles/taiga-ui-global.less",
+              "node_modules/@taiga-ui/styles/taiga-ui-global.less",
```

Comment:
We should probably prompt people if they want to install global styles.

Reply:
fixed